ZenPacks/zenoss/ZenPackLib/lib/base/ComponentBase.py

```diff
@@ -231,6 +231,13 @@ def get_containing_relname(self):
         """Return name of containing relationship."""
         for relname, relschema in self._relations:
             if issubclass(relschema.remoteType, ToManyCont):
+                # skip if it's not the real containing rel
+                # for instance an inherited os.filesystems for sub-classed FileSystem
+                rel = getattr(self, relname, None)
```

Comment:
Can you explain this a bit more?  How is that not a "real" containing relationship?